views/clucks/clucksIndex.ejs

```diff
@@ -0,0 +1,37 @@
+<%- include("../partials/header") %> 
+<div style="text-align:center">
+<% if (username) { %>
+<h1>HI <%= username.toUpperCase() %>, <br>WELCOME TO CLUCKR 🐓</h1>
+<% } else { %>
+<h1> Please sign in to Cluckr. </h1>
+<% } %>
+</div>
+
+<% if (username) { %>
+
+<form action="/sign_out" method="POST" class="form-inline">
```

Comment:
Don't forget to indent all code withing `if - else` blocks. Proper indentation will be your friend.